doc/api/yonomi-sdk/ThermostatRepository/getThermostatTrait.md

```diff
@@ -40,6 +40,8 @@ static ThermostatTrait getThermostatTrait(dynamic trait) {
       FanMode(trait.state.fanMode.reported?.value ?? AvailableFanMode.ON),
       ThermostatMode(
           trait.state.mode.reported?.value ?? AvailableThermostatMode.OFF),
+      AmbientTemperature(
+          trait.state.ambientTemperature.reported?.value ?? 0.0),
```

Comment:
Should this default to 0.0? Seems like we might want it to default to null, as 0.0 means something pretty specific.